clients/src/main/java/org/apache/kafka/common/resource/ResourceType.java

```diff
@@ -68,7 +68,12 @@ public enum ResourceType {
     /**
      * A user principal
      */
-    USER((byte) 7);
+    USER((byte) 7),
+
+    /**
+     *  Client metrics
+     */
+    CLIENT_METRICS((byte) 8);
```

Comment:
The KIP does not define this constant. Checking is against the `CLUSTER` resource type.

Reply:
Missed this thing, thanks. Removed and corrected tests.